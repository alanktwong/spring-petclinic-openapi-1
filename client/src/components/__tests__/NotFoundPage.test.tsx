import React from 'react';
import renderer from 'react-test-renderer';
import NotFoundPage from '../NotFoundPage';

it('renders correctly', () => {
  const tree = renderer.create(<NotFoundPage />).toJSON();
  expect(tree).toMatchSnapshot();
});
