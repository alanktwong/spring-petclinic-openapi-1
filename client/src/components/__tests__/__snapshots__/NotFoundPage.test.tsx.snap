// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h3>
  The requested page has not been found.
</h3>
`;
